migrations/v2.js

```diff
@@ -0,0 +1,91 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+const getCourse = content => {
+  const course = content.find(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._extensions?._glossary;
+};
+
+describe('Glossary - v2.0.3 to v2.1.0', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-glossary/compare/v2.0.3..v2.1.0
+
+  let course;
+
+  whereFromPlugin('Glossary - from v2.0.3', { name: 'adapt-contrib-glossary', version: '<2.1.0' });
```

Comment:
Recent change just to prevent it migrating anything lower than v2.
```suggestion
  whereFromPlugin('Glossary - from v2.0.3', { name: 'adapt-contrib-glossary', version: '>=2.0.0 <2.1.0' });
```